### PR TITLE
Single contact - show contact name

### PR DESCRIPTION
**Pull request by brianteeman**
The option to display the contact name was inside the "contact information" options and was only visible to set if the "contact information" was set to "show"

It shouldn't be connected to the "contact information" as the display of the contact name is not conditional on the display of the contact information

This simple PR moves the field out of the contact information block and condition

Created this PR as it took me ages to find the option to remove the contact name from the page as the option was hidden ;)

### CODE REVIEW

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 85d41663534ccf16a6bc6654e5cd915974c5c587<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by kofaysi** [comment]
I have tested this item :red_circle: unsuccessfully on 85d41663534ccf16a6bc6654e5cd915974c5c587<br /><br />I do not see any change in option order (Global Configuration > Contacts) after applying the patch on J 3.9.2
[contact info hide](https://screenshots.firefox.com/7FejtgTaPsrLVLT1/svjsvetova523-1.cz "contact info hide")
[contact info show](https://screenshots.firefox.com/C8cgAiZJWMYCTqe0/svjsvetova523-1.cz "contact info show")<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by kofaysi** [comment]
I do not see any change in order of the options (Global configuration > Contacts) after applying this patch.
[contact info hide](https://screenshots.firefox.com/7FejtgTaPsrLVLT1/svjsvetova523-1.cz "contact info hide")
[contact info show](https://screenshots.firefox.com/C8cgAiZJWMYCTqe0/svjsvetova523-1.cz "contact info show")<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by ladyjer** [comment]
I have tested this item :red_circle: unsuccessfully on 85d41663534ccf16a6bc6654e5cd915974c5c587<br /><br />Hi,

tested succesfully for category/tmp/default.xml and contact/tmp/deault.xml.
tested unsucessfully for **categories**/tmpl/default.xml. Order now is ok but still present "showon="show_info:1" for name field.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by ladyjer** [comment]
Hi,

tested succesfully for category/tmp/default.xml and contact/tmp/deault.xml.
tested unsucessfully for **categories**/tmpl/default.xml. Order now is ok but still present "showon="show_info:1" for name field.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by brianteeman** [comment]
bugs and conflicts fixed

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 45cf4e77953714c8c6b0e654130099bf4ecc7e65<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by ladyjer** [comment]
I have tested this item :white_check_mark: successfully on 45cf4e77953714c8c6b0e654130099bf4ecc7e65<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by kofaysi** [comment]
I have tested this item :red_circle: unsuccessfully on 45cf4e77953714c8c6b0e654130099bf4ecc7e65<br /><br />I do not see any change in option order (Global Configuration > Contacts) after applying the patch on J3.9.3.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by ladyjer** [comment]
It's not a matter of Global Configuration > Contacts.
The issue is about option on menu item on contacts/contact category/contact categories<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by kofaysi** [comment]
I have tested this item :white_check_mark: successfully on 45cf4e77953714c8c6b0e654130099bf4ecc7e65<br /><br />Ok, I see now. What is the reason for
* leaving the other contact details (Contact's Position, Email, Street Address, City or Suburb, State or County, Postal/ZIP Code, Country, Telephone, Mobile Phone, Fax, Webpage, Image) - or at least dealing with bulks: Position, Email, Address, Phones, Webpage; **or**
* not changing the visible also in the Global configuration if
> It shouldn't be connected to the "contact information" as the display of the contact name is not conditional on the display of the contact information<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23563">issues.joomla.org/tracker/joomla-cms/23563</a>.</sub>

**Comment by HLeithner** [comment]
thx

**Comment by brianteeman** [comment]
thanks

File: components/com_contact/views/categories/tmpl/default.xml [hunk 1]
```diff
@@ -399,23 +399,22 @@
...
-				name="show_tags"
+			<field 
+				name="show_name"
 				type="list"
-				label="COM_CONTACT_FIELD_SHOW_TAGS_LABEL"
-				description="COM_CONTACT_FIELD_SHOW_TAGS_DESC"
+				label="COM_CONTACT_FIELD_PARAMS_NAME_LABEL"
+				description="COM_CONTACT_FIELD_PARAMS_NAME_DESC"
 				useglobal="true"
-				class="chzn-color"
```
Comment (wilsonge) [inline components/com_contact/views/categories/tmpl/default.xml:408]: dont' think you meant to remove this class

**Pull request merged**